docs/ota-client-guide/modules/ROOT/pages/enable-device-cred-provisioning.adoc

```diff
@@ -6,13 +6,18 @@ Once you're ready to provision devices in production, you need to build disk ima
 
 After you have flashed those images to devices, you boot the image and install the device certicate for each device. You can install the certificate to the device's fileystem or use an HSM.
```

Comment:
This needs to have a sentence added along the lines of "You can also have your HSM generate a private key and certificate on the device, then sign the certificate with your fleet root key--but we don't have docs on how to do that, so for these instructions we assume you're generating the device certificates and private keys on your development computer and copying them over."